synapse/handlers/sso.py

```diff
@@ -195,6 +197,8 @@ def __init__(self, hs: "HomeServer"):
         self._error_template = hs.config.sso.sso_error_template
         self._bad_user_template = hs.config.sso.sso_auth_bad_user_template
         self._profile_handler = hs.get_profile_handler()
+        self._media_repo = hs.get_media_repository()
+        self._http_client = hs.get_proxied_http_client()
```

Comment:
Change this to use `get_proxied_blacklisted_http_client` please.

Reply:
Done!